setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import setup
+
+setup(
+    name="simpleocr",
+    packages=["simpleocr"],
+    version="1.0.0",
+    description="A library for simple OCR in Python using OpenCV",
+    author="The simple-ocr-opencv authors",
+    url="https://www.github.com/goncalopp/simple-ocr-opencv",
+    download_url="https://www.github.com/goncalopp/simple-ocr-opencv/archive/1.0.0.tar.gz",
```

Comment:
Is there a strong advantage to have `download_url`? It's yet one more place where we have to maintain the correct version numbers, and I expect most if not all of the "stable" downloads would happen on pypi. If there isn't a strong incentive, I'd just remove it